config/symbols.us.bomar.txt

```diff
@@ -1,3 +1,14 @@
 actor_names = 0x80180680;
 MAR_PrizeDrops = 0x80180744;
+bird_cage_pos_x = 0x80181280;
+bird_cage_pos_y = 0x80181284;
+statue_pos_x_1 = 0x80181288;
+statue_pos_x_2 = 0x8018128C;
+gear_pos_x = 0x80181290;
+stone_door_pos_x_1 = 0x80181294;
+stone_door_pos_x_2 = 0x80181298;
+statue_pos_x_3 = 0x801812A0;
+anim_bird_cage = 0x801812A8;
+anim_gear_1 = 0x801812C4;
+anim_gear_2 = 0x801812D0;
```

Comment:
Renamed these just as placeholders - can replace these with better names once the data is extracted, as there is some overlapping array data etc.